web-wallet/src/lib/components/Send/Send.svelte

```diff
@@ -1,337 +1,337 @@
-<svelte:options immutable={true}/>
+<svelte:options immutable={true} />
 
 <script>
-	import { fade } from "svelte/transition";
-	import { createEventDispatcher, onMount } from "svelte";
-	import { mdiArrowUpBoldBoxOutline, mdiWalletOutline } from "@mdi/js";
-
-	import { deductLuxFeeFrom } from "$lib/contracts";
-	import { duskToLux, luxToDusk } from "$lib/dusk/currency";
-	import { validateAddress } from "$lib/dusk/string";
-	import { logo } from "$lib/dusk/icons";
-	import {
-		AnchorButton,
-		Badge,
-		Button,
-		Icon,
-		Textbox,
-		Wizard,
-		WizardStep
-	} from "$lib/dusk/components";
-	import {
-		ContractStatusesList,
-		GasFee,
-		GasSettings,
-		OperationResult,
-		ScanQR
-	} from "$lib/components";
-
-	/** @type {(to: string, amount: number, gasPrice: number, gasLimit: number) => Promise<string>} */
-	export let execute;
-
-	/** @type {(amount: number) => string} */
-	export let formatter;
-
-	/** @type {ContractGasSettings} */
-	export let gasSettings;
-
-	/** @type {number} */
-	export let spendable;
-
-	/** @type {ContractStatus[]} */
-	export let statuses;
-
-	/** @type {number} */
-	let amount = 1;
-
-	/** @type {string} */
-	let address = "";
-
-	/** @type {import("qr-scanner").default} */
-	let scanner;
-
-	/** @type {import("..").ScanQR} */
-	let scanQrComponent;
-
-	/** @type {HTMLInputElement | null} */
-	let amountInput;
-
-	/** @type {boolean} */
-	let isValidGas = true;
-
-	/** @type {boolean} */
-	let isNextButtonDisabled = false;
-
-	let { gasLimit, gasPrice } = gasSettings;
-
-	const minAmount = 0.000000001;
-
-	const dispatch = createEventDispatcher();
-	const resetOperation = () => dispatch("operationChange", "");
-
-	onMount(() => {
-		amountInput = document.querySelector(".operation__input-field");
-	});
-
-	$: luxFee = gasLimit * gasPrice;
-	$: fee = formatter(luxToDusk(luxFee));
-	$: maxSpendable = deductLuxFeeFrom(spendable, luxFee);
-	$: isAmountValid = amount >= minAmount && amount <= maxSpendable;
-	$: totalLuxFee = luxFee + duskToLux(amount);
-	$: isFeeWithinLimit = totalLuxFee <= duskToLux(spendable);
-	$: isNextButtonDisabled = !(isAmountValid && isValidGas && isFeeWithinLimit);
-
-	$: addressValidationResult = validateAddress(address);
+  import { fade } from "svelte/transition";
+  import { createEventDispatcher, onMount } from "svelte";
+  import { mdiArrowUpBoldBoxOutline, mdiWalletOutline } from "@mdi/js";
+
+  import { deductLuxFeeFrom } from "$lib/contracts";
+  import { duskToLux, luxToDusk } from "$lib/dusk/currency";
+  import { validateAddress } from "$lib/dusk/string";
+  import { logo } from "$lib/dusk/icons";
+  import {
+    AnchorButton,
+    Badge,
+    Button,
+    Icon,
+    Textbox,
+    Wizard,
+    WizardStep,
```

Comment:
Looks like this extra comma is added if there are more than 2 or 3 items in between the curly brackets